### DISCUSSION TITLE
Welcome screen with possibility to set theme and font family

### DISCUSSION THREAD

**Discussion by SenseiMarv** [ghostty-org/ghostty#3743] [Feature Requests, Ideas] [answered]
I have just started to try out Ghostty and it has made a good impression so far. I like the attempt with the "Zero Configuration Philosophy", but at least in my mind there will always be two things that users will most likely want to change to their liking: the theme and the font family.

I think it's worth discussing adding some sort of "welcome screen" where the user is greeted with a little introduction, e.g. common hotkeys, and then given the option to choose a theme and font family, with the current default selected.

**Comment by pluiedev** [comment, marked as answer]
I think it's important that when a user downloads a terminal, it works like a terminal. Nobody likes it when the likes of Warp forces you to sit through several screens of login and configuration and so on, wasting minutes of your time before you actually get to use the terminal. If they do find it necessary to configure Ghostty, then they can find it very easily via the idiomatic hamburger menu or by standardized keybinds like `ctrl/cmd+,` (when #2354 is merged, of course).

**Reply by 00-kat** [reply]
Also, Mitchell has previously rejected onboarding screens too, as he personally does not like them.
